src/test/java/org/opentripplanner/routing/algorithm/mapping/BikeRentalSnapshotTest.java

```diff
@@ -109,8 +110,11 @@ public void directBikeRental() {
 
         try {
             expectArriveByToMatchDepartAtAndSnapshot(request);
-        } catch (CompletionException e) {
-            RoutingValidationException.unwrapAndRethrowCompletionException(e);
+            throw new IllegalArgumentException();
+        } catch (Exception e) {
+            // for some strange reason the stack trace actually is not printed to the console
+            e.printStackTrace();
+            Arrays.stream(e.getSuppressed()).forEach(Throwable::printStackTrace);
```

Comment:
What is going on here? I do not understand why the IAE is thrown or who this test can fail in the case there is something wrong.